charts/amazon-cloudwatch-observability/templates/linux/fluent-bit-configmap.yaml

```diff
@@ -17,27 +17,27 @@ data:
 {{- if hasPrefix "us-iso-" .Values.region }}
```

Comment:
not an expert with FB config but what happens if a custom fluentbit config doesn't have `Parsers_File` for some reasons? would it be better to just appned right after `[SERVICE]`?

Reply:
what do you mean custom config is out of scope? the public doc should have some instructions on how to setup ipv6 with custom configs, right? The new function `fluent-bit.add-ipv6-preference` will skip if a custom config already has dual stack configred.